packages/dataviews/src/dataforms-layouts/panel/index.tsx

```diff
@@ -98,6 +99,22 @@ function PanelDropdown< Item >( {
 		};
 	}, [ field ] );
 
+	const fieldValue = useMemo( () => {
+		if ( Array.isArray( data ) ) {
+			const [ firstRecord, ...remainingRecords ] = data;
+			const firstValue = fieldDefinition.getValue( {
+				item: firstRecord,
+			} );
+			const intersects = remainingRecords.every( ( item ) => {
+				return fieldDefinition.getValue( { item } ) === firstValue;
+			} );
+			return intersects ? firstValue : MIXED_VALUE;
+		}
+		return fieldDefinition.getValue( {
+			item: data,
+		} );
+	}, [ data, fieldDefinition ] );
```

Comment:
If I'm not wrong this merging is probably duplicated in "regular" too. Should we extract it to a utility?

Reply:
Correct, I can move it to a utility 👍